apps/api/.env.example

```
@@ -9,6 +9,9 @@ DB_DATABASE=application_ctx
 REDIS_HOST=redis
 REDIS_PORT=6379
 
+DRAINING_RUNNER_CONCURRENCY=2
+DRAINING_BOX_CONCURRENCY=10
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DRAINING_BOX_CONCURRENCY key should go before the DRAINING_RUNNER_CONCURRENCY key

(UnorderedKey)